app/electron/main.ts

```diff
@@ -453,6 +453,78 @@ class PluginManagerEventListeners {
   }
 }
 
+// getShell returns the shell to use.
+function getShell(): string {
+  if (process.platform === 'win32') {
+    // In Windows, COMSPEC is the default shell
+    return process.env.COMSPEC || 'cmd.exe';
+  }
+
+  let shell: string;
+  try {
+    shell = userInfo().shell || process.env.SHELL || '';
+  } catch (error) {
+    console.error('Failed to get user shell:', error);
+    shell = process.env.SHELL || '';
+  }
+
+  // Fallback chain: user's shell > SHELL env > zsh > bash > sh
+  if (!shell || shell.trim() === '') {
+    shell = '/bin/zsh';
+  }
+  if (!fs.existsSync(shell)) {
+    // Fallback to bash if zsh is not found
+    console.error('Shell not found:', shell);
+    shell = '/bin/bash';
```

Comment:
A better default would be `/bin/sh`. Because that's a default others use, and is more likely to be there because bash is optional.